test/validation/api/system/system.c

```diff
@@ -277,7 +277,8 @@ static void system_test_meminfo(void)
 	CU_ASSERT(ret == ret_0);
 	CU_ASSERT(info_0.total_mapped == info.total_mapped);
 	CU_ASSERT(info_0.total_used == info.total_used);
-	CU_ASSERT(info_0.total_overhead == info.total_overhead);
+	if (info_0.total_overhead)
```

Comment:
The condition does not seem necessary. Even if total_overhead is not supported, its value (zero in that case) should be the same in the two calls.

Reply:
What I was thinking here was that an implementation could still fill `total_overhead` when it has to read the per memory block info anyways. But now thinking about it, perhaps  it's not the cleanest if the field value depends on other parameters.

Reply:
The "zero if not supported by the implementation" text in the API makes me think that if total overhead count is supported, then it is returned in all calls. If it might be zero also (e.g.) when the return value is less than the num parameter, then I think the API probably should mention it.